alfresco-transform-imagemagick/alfresco-transform-imagemagick-boot/src/test/java/org/alfresco/transformer/ImageMagickControllerTest.java

```diff
@@ -228,7 +227,7 @@ public void cropGravityGoodTest() throws Exception
     {
         for (String value : new String[]{"North", "NorthEast", "East", "SouthEast", "South", "SouthWest", "West", "NorthWest", "Center"})
         {
-            expectedOptions = "-gravity " + value + " +repage";
+            expectedOptions = "-auto-orient " + "-gravity " + value + " +repage";
```

Comment:
Was "-auto-orient" required for metadata extraction or some other test fix ?

Reply:
I spotted that it was missing in the T-Engine. I'm sure I passed this bug fix on to your team during Paddington. This was bug was introduced in the first version of the T-Engine (included in ACS 6.0). Found on the repo side in new test while adding bidirectional conversion between TransformationOptions and Map<String,String> for transform options.